ml-agents/mlagents/trainers/torch/attention.py

```diff
@@ -82,105 +101,85 @@ def forward(
         return value_attention, att
```

Comment:
Do we have a reason to be returning the attention matrix?

Reply:
not really, but it is left as output in a lot of implementations I found online. It does not cost anything to return it.